modules/components/pages/outputs/snowflake_streaming.adoc

```diff
@@ -279,13 +279,13 @@ init_statement: |2
 
 === `schema_evolution`
 
-Options to control schema updates as new columns are added to the pipeline.
+Options to control schema updates as new tables and columns are added to the pipeline.
 
 *Type*: `object`
 
 === `schema_evolution.enabled`
 
-Whether schema evolution is enabled.
+Whether schema evolution is enabled. Set to `true` to allow the automatic creation of new tables and columns in the <<schema,specified schema>> when they are added to your pipeline.
```

Comment:
Given , we might need more here:

> If your pipeline results in new tables or fields (columns), these additions will automatically trigger the schema evolution process to create corresponding tables or columns in the specified schema, provided schema evolution is enabled.

Reply:
@rockwotj - Are you happy with this update?

Reply:
* I don't think the "provided schema evolution is enabled." is needed
* To be clear a single output can only create the output table. Maybe saying multiple tables can be created is confusing?

Reply:
How I would say it:

> Whether schema evolution is enabled. When set to true, the table will be created based on the schema in the first message written if it doesn't already exist. As new fields are added to the messages in the pipeline, then existing columns in the Snowflake table will be created. Required columns will be marked as `nullable` if new messages don't contain data for them.


Here are the snowflake docs if that's helpful, but theirs works slightly different
https://docs.snowflake.com/en/user-guide/data-load-snowpipe-streaming-kafka-schema-detection#usage-notes

Reply:
Updated